tests/e2e/deployment/deployment_test.go

```diff
@@ -194,4 +207,103 @@ var _ = Describe("Application deployment test in E2E scenario", func() {
 			utils.CheckPodDeleteState(ctx.Cfg.K8SMasterForKubeEdge+constants.AppHandler, podlist)
 		})
 	})
+
+	Context("StatefulSet lifecycle test in edge node", func() {
+		BeforeEach(func() {
+			// Get current test SpecReport
+			testSpecReport = CurrentSpecReport()
+			// Start test timer
+			testTimer = DeploymentTestTimerGroup.NewTestTimer(testSpecReport.LeafNodeText)
+		})
+
+		AfterEach(func() {
+			// End test timer
+			testTimer.End()
+			// Print result
+			testTimer.PrintResult()
+
+			By(fmt.Sprintf("get StatefulSet %s", UID))
+			deployment, err := utils.GetStatefulSet(clientSet, metav1.NamespaceDefault, UID)
+			Expect(err).To(BeNil())
+
+			By(fmt.Sprintf("list pod for StatefulSet %s", UID))
+			labelSelector := labels.SelectorFromSet(map[string]string{"app": UID})
+			_, err = utils.ListPods(clientSet, metav1.NamespaceDefault, labelSelector, nil)
+			Expect(err).To(BeNil())
+
+			By(fmt.Sprintf("delete StatefulSet %s", UID))
+			err = utils.DeleteStatefulSet(clientSet, deployment.Namespace, deployment.Name)
+			Expect(err).To(BeNil())
+
+			By(fmt.Sprintf("wait for pod of StatefulSet %s disappear", UID))
+			err = utils.WaitForPodsToDisappear(clientSet, metav1.NamespaceDefault, labelSelector, constants.Interval, constants.Timeout)
+			Expect(err).To(BeNil())
+
+			utils.PrintTestcaseNameandStatus()
+		})
+
+		It("Basic StatefulSet test", func() {
+			replica := int32(2)
```

Comment:
it this replica designed?

Reply:
yes

Reply:
user always have multi replicas when use statefulset

Reply:
/lgtm